readme.md

```diff
@@ -6,11 +6,13 @@ Project Apicula uses a combination of fuzzing and parsing of the vendor data fil
 
 ##  Dependencies
 
-Version 1.9 of the Gowin vendor tools, and a copy of the Excel pinout files downloaded in `~/Documents/gowinsemi`
+Version 1.9.1.01 of the Gowin vendor tools, and a copy of the Excel pinout files downloaded in `~/Documents/gowinsemi`:
+* UG107-1.07E_GW1N-1 Pinout.xlsx
+* UG801-1.5E_GW1NR-9 Pinout.xlsx
```

Comment:
I tested it with 1.9.3 now, so any 1.9 release should work.
Maybe small nitpick, but I'd write "copy of the following"
Also worth noting: I used an older version of the excel files, so the names change sometimes.

I'm a bit torn. Might be good to specify exact versions for reliability,  but it really should work with any recent version.

Reply:
I would recommend specifying what you use. If someone has luck with other versions then it's a secondary win.